bahmni_api_feed/models/api_event_worker.py

```diff
@@ -94,7 +94,7 @@ def _get_address_details(self, address):
 
     @api.model
     def _find_country(self, address):
-        return self.env['res.company'].search([], limit=1).partner_id.country_id
+        return self.env['res.country'].sudo().search([('name', '=', address.get('country'))], limit=1)
```

Comment:
@hari1119 Please do not use sudo, it is removed now and we are passing sessionid

Reply:
okey @mohan-13